test/e2eTests/pages/page.js

```diff
@@ -34,11 +47,19 @@ class Page {
 
   /** Validates that the page is loaded */
   isLoaded() {
-    this.I.waitForElement(this.options.isLoaded.element, 5);
```

Comment:
Are there any pages that have the same url bu different elements? If so, we still want this. I can't remember why we added it in the first place, but it may have been for the above reason. It also may have just been to verify the page was loaded, and in that case the waitForNavigation is a much better solution.